src/train_anger_classifier.py

```diff
@@ -0,0 +1,14 @@
+import pickle
```

Comment:
Is this whole file intended to be checked in? it seems like a new feature, beyond the PR description, and doesn't seem to be referenced anywhere else (like a bazel build)

Reply:
Oh! I forgot to mention it in the PR description. I added it because Naveen's old anger words classifier no longer works with the newer version of libraries. I found his code for training the classifier. It only needs to be executed once, but I think it'd be nice to have it in the repo, just in case in the future there are other compatibility issues.